.env.sample

```diff
@@ -1,69 +1,56 @@
+## ENVIRONMENT production|development
 NODE_ENV=development
-PORT=3000
-WEB_PORT=3001
-WEB_EXPOSED_PORT=443
-WEB_EXPOSED_HTTP_PORT=80
-HOST_HOSTNAME=localhost
-WEB_URL=http://localhost:3001
-REACT_APP_PROXY_URL=http://localhost:3000
-
-## Database Details
-DB_USERNAME=
-DB_PASSWORD=
-DB_PORT=3306
-DB_NAME=tombolo
-DB_HOSTNAME=mysql_db
-
-## Auth Service details
-#For authentication, Tombolo uses AuthService module, which needs to be setup separetly.
-AUTH_SERVICE_URL=<protocol>://<host_name>:<port>/api/auth
-AUTHSERVICE_TOMBOLO_CLIENT_ID=
-secret=
-
-## SSL certificate for the Web Application
-CERT_PATH=
 
-#used to show/hide AD Search in Collaborator module
-REACT_APP_LDAP_SEARCH_ENABLED=false
+## HOST localhost|hostname
+HOST_HOSTNAME=
 
-#Kafka Set up - optional
-#To create keys, please refer here https://github.com/edenhill/librdkafka/wiki/Using-SSL-with-librdkafka#create-a-ca-certificate
-#For documentation on bitnami kafka and zookeeper docker images and setup, please refer https://github.com/bitnami/bitnami-docker-kafka, https://github.com/bitnami/bitnami-docker-zookeeper
-#For details on SSL config https://github.com/bitnami/bitnami-docker-kafka/issues/129
-#Kafka groupId
-KAFKA_PORT=9092
-KAFKA_EXTERNAL_PORT=29092
-KAFKA_MGR_PORT=9000
-KAFKA_ADVERTISED_LISTENER=
-ZK_PORT=2181
-KAFKA_HOST_NAME=
-KAFKA_CLIENT_USER=
-KAFKA_CLIENT_PASSWORD=
-KAFKA_CERTIFICATE_PASSWORD=
-
-## Zoo Keeper setup
-ZOO_SERVER_USERS=
-ZOO_SERVER_PASSWORDS=
-ZOO_CLIENT_USER=
-ZOO_CLIENT_PASSWORD=
-ZOO_TLS_CLIENT_KEYSTORE_FILE=/opt/bitnami/kafka/conf/certs/zookeeper.keystore.jks
-ZOO_TLS_CLIENT_KEYSTORE_PASSWORD=
-ZOO_TLS_CLIENT_TRUSTSTORE_FILE=/opt/bitnami/kafka/conf/certs/zookeeper.truststore.jks
-ZOO_TLS_CLIENT_TRUSTSTORE_PASSWORD=
+## BACKEND PORT
+PORT=3000
 
-JOB_COMPLETE_TOPIC=
-START_JOB_TOPIC
-JOB_COMPLETE_GROUP_ID=tombolo
-JOB_COMPLETE_TOPIC=JobComplete
+## MYSQL SSL true|false
+MYSQL_SSL_ENABLED=
 
-## Real BI
-REALBI_URL=
+## NGINX SSL relative path|empty
+CERT_PATH=
```

Comment:
gives me error when I try to build container with empty volume, maybe lets keep certs/ as a placeholder